docs/docsite/rst/user_guide/playbooks_reuse_roles.rst

```diff
@@ -149,9 +175,12 @@ Or, using the newer syntax::
           vars:
             dir: '/opt/a'
             app_port: 5000
+          tags: typeA
       ...
 
-You can conditionally import a role and execute its tasks::
+When you add a :ref:`tag <tags>` to an ``include_role`` task, Ansible applies the tag ONLY to the include itself. This means you can pass ``--tags`` to run only selected tasks from the role, if those tasks themselves have the same tag as the include statement. See :ref:`selective_reuse` for details.
```

Comment:
I tend to avoid using all caps for emphasis and instead use bold.

Reply:
IBM Style calls for italics to connote emphasis.

Reply:
In that case, italics it should be.